### PR TITLE
Scrom fix cbp

### PR DESCRIPTION
**Pull request by bharathravi-in**

### CODE REVIEW

File: project/ws/viewer/src/lib/plugins/html/html.component.ts [hunk 1]
```diff
@@ -358,26 +358,30 @@ export class HtmlComponent implements OnInit, OnChanges, OnDestroy {
...
+                // `${environment.azureHost}/${environment.azureBucket}/content/html/${this.htmlContent.identifier}-snapshot/index.html?timestamp='${new Date().getTime()}`
+                // tslint:disable-next-line: max-line-length
+                `${environment.azureHost}/${environment.azureBucket}/content/html/${this.htmlContent.identifier}-snapshot/index.html?timestamp='${new Date().getTime()}`
+              )
+            } else {
+              this.iframeUrl = this.domSanitizer.bypassSecurityTrustResourceUrl(
+                // tslint:disable-next-line: max-line-length
+                // `${environment.azureHost}/${environment.azureBucket}/content/html/${this.htmlContent.identifier}-snapshot/index.html?timestamp='${new Date().getTime()}`
+                // tslint:disable-next-line: max-line-length
+                `/abcd/${environment.azureBucket}/content/html/${this.htmlContent.identifier}-snapshot/index.html?timestamp='${new Date().getTime()}`
```
Comment (vishnubansaltarento) [inline project/ws/viewer/src/lib/plugins/html/html.component.ts:384]: abcd , we need to add?

File: project/ws/viewer/src/lib/plugins/html/html.component.ts [hunk 2]
```diff
@@ -358,26 +358,30 @@ export class HtmlComponent implements OnInit, OnChanges, OnDestroy {
...
-        if (this.htmlContent.streamingUrl && this.htmlContent.initFile) {
-          this.iframeUrl = this.domSanitizer.bypassSecurityTrustResourceUrl(
-            // tslint:disable-next-line:max-line-length
-            `${this.generateUrl(this.htmlContent.streamingUrl)}/${this.htmlContent.initFile}?timestamp='${new Date().getTime()}`
-          )
+        if (this.htmlContent.streamingUrl.includes(environment.azureHost)) {
+          this.iframeUrl = this.domSanitizer.bypassSecurityTrustResourceUrl(this.htmlContent.streamingUrl)
         } else {
-          if (environment.production) {
+          if (this.htmlContent.streamingUrl && this.htmlContent.initFile) {
```
Comment (vishnubansaltarento) [inline project/ws/viewer/src/lib/plugins/html/html.component.ts:366]: put additional check for this.htmlContent

File: project/ws/viewer/src/lib/plugins/html/html.component.ts [hunk 3]
```diff
@@ -358,26 +358,30 @@ export class HtmlComponent implements OnInit, OnChanges, OnDestroy {
...
-              // `${environment.azureHost}/${environment.azureBucket}/content/html/${this.htmlContent.identifier}-snapshot/index.html?timestamp='${new Date().getTime()}`
-              // tslint:disable-next-line: max-line-length
-              `/abcd/${environment.azureBucket}/content/html/${this.htmlContent.identifier}-snapshot/index.html?timestamp='${new Date().getTime()}`
-            )
+            if (environment.production) {
+              this.iframeUrl = this.domSanitizer.bypassSecurityTrustResourceUrl(
+                // tslint:disable-next-line: max-line-length
+                // `${environment.azureHost}/${environment.azureBucket}/content/html/${this.htmlContent.identifier}-snapshot/index.html?timestamp='${new Date().getTime()}`
+                // tslint:disable-next-line: max-line-length
+                `${environment.azureHost}/${environment.azureBucket}/content/html/${this.htmlContent.identifier}-snapshot/index.html?timestamp='${new Date().getTime()}`
```
Comment (vishnubansaltarento) [inline project/ws/viewer/src/lib/plugins/html/html.component.ts:377]: put additional check for this.htmlContent

File: project/ws/viewer/src/lib/plugins/html/html.component.ts (hunk 1 above)
Comment (bharathravi-in) [reply project/ws/viewer/src/lib/plugins/html/html.component.ts:384]: this is used for local proxy it is needed

File: project/ws/viewer/src/lib/plugins/html/html.component.ts (hunk 2 above)
Comment (bharathravi-in) [reply project/ws/viewer/src/lib/plugins/html/html.component.ts:366]: ok adding on top line no 347

File: project/ws/viewer/src/lib/plugins/html/html.component.ts (hunk 3 above)
Comment (bharathravi-in) [reply project/ws/viewer/src/lib/plugins/html/html.component.ts:377]: added on top line no 347

**Pull request merged**